filter-senderscore.go

```diff
@@ -130,27 +130,31 @@ func filterConnect(phase string, sessionId string, params[] string) {
 	}
 }
 
+func printMsg(msgType string, sessionId string, token string, format string, a ...interface{}) {
+	var args []interface{}
+
+	if version < "0.5" {
+		args = append([]interface{}{msgType, token, sessionId}, a...)
+	} else {
+		args = append([]interface{}{msgType, sessionId, token}, a...)
+	}
+
+	fmt.Printf("%s|%s|%s|" + format + "\n", args...)
```

Comment:
Alternatively, we could make this three individual calls to `Printf` and avoid messing with the format string and argument list.

Reply:
Or two calls to `Printf` and one additional line to print the newline character, to be more specific.